charmcraft/manifest.py

```diff
@@ -32,33 +32,22 @@
 def create_manifest(
     basedir: pathlib.Path,
     started_at: datetime.datetime,
-    bases_config: Optional[config.BasesConfiguration] = None,
+    bases_config: Optional[config.BasesConfiguration],
 ):
     """Create manifest.yaml in basedir for given base configuration.
 
     :param basedir: Directory to create Charm in.
     :param started_at: Build start time.
-    :param bases_config: Relevant bases configuration.
+    :param bases_config: Relevant bases configuration, if any.
 
     :returns: Path to created manifest.yaml.
     """
-    if bases_config is None:
-        os_platform = utils.get_os_platform()
-
-        # XXX Facundo 2021-03-29: the architectures list will be provided by the caller when
-        # we integrate lifecycle lib in future branches
-        architectures = [utils.get_host_architecture()]
+    content = {
+        "charmcraft-version": __version__,
+        "charmcraft-started-at": started_at.isoformat() + "Z",
+    }
 
-        name = os_platform.system.lower()
-        channel = os_platform.release
-        bases = [
-            {
-                "name": name,
-                "channel": channel,
-                "architectures": architectures,
-            }
-        ]
-    else:
+    if bases_config is not None:
```

Comment:
Please add a comment here that this may be the case for bundles, thanks!

Reply:
Will do

Reply:
Done!